modules/tinyexr/SCsub

```diff
@@ -21,3 +21,10 @@ env_thirdparty.add_source_files(env.modules_sources, thirdparty_sources)
 
 # Godot's own source files
 env_tinyexr.add_source_files(env.modules_sources, "*.cpp")
+
+if not env.msvc:
+	env_tinyexr.Append(CCFLAGS=['-fno-lto'])
```

Comment:
Note that it couldn't work, you're only disabling LTO on `env_tinyexr` (i.e. the files in `modules/tinyexr/*.cpp`) and not on the tinyexr thirdparty code itself, which is included above in the separate `env_thirdparty`.

But I had tried to disable LTO in `env_thirdparty` already, with the results documented in #24641.

Reply:
Might be worth a try to disable LTO in both envs though (or `env_tinyexr` before it's cloned in `env_thirdparty`).